sources.json

```diff
@@ -242,6 +242,78 @@
     "tags": ["odata", "promise"],
     "license": "mit"
   },
+  {
+    "owner": "mauriciolauffer",
+    "repo": "https://github.com/mauriciolauffer/openui5-error-collector",
+    "addedToBoUI5": "2022-07-10T11:52:00.000Z",
+    "type": "customControl",
```

Comment:
@mauriciolauffer @marianfoo : looks more like a `module`?

Reply:
Yep, it makes sense, if `module` means everything that isn't a customControl.

IMO, we should improve documentation and describe each of the enum entries to avoid this same issue:
https://github.com/ui5-community/bestofui5-data/blob/5860fd33a980bcdc8c23fb3b7bf25d7c36607ebe/src/types.d.ts#L1-L8

Reply:
you´re right, working on it https://github.com/ui5-community/bestofui5-data/issues/56